travel/Html/travel_platform.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```

Comment:
The main.html is a block for inserting into base.html in Django.
The travel_platform is full html, as it was developed.